data_prep/graph_dataset.py

```diff
@@ -35,7 +36,10 @@ def __init__(self, corpus):
         all_docs = ['doc.{}'.format(i) for i in range(len(self._raw_texts))]
 
         print('Preprocess corpus')
-        tokenized_text, self._tokens = self._preprocess()
+        if type(corpus) == IMDbData:
+            tokenized_text, self._tokens = self._preprocess(lower_threshold=10, upper_threshold=30)
+        else:
+            tokenized_text, self._tokens = self._preprocess()
```

Comment:
You could make this if/else a one liner with something like `lower_threshold, upper_threshold = (10, 30) if type(corpus) == IMDbData else (4, 50)` I guess or similar, but this is optional from my side

Reply:
Ok, I can do that and then merge. Btw the thresholds and window size are mentioned in the appendix